docs/_docs/security/sandbox.adoc

```diff
@@ -6,26 +6,75 @@ This user-defined code can utilize Java features to get access to host resources
 For example, it can create/update/delete files; create sockets; read/update/delete system properties; use reflection API and so on.
 To restrict user-defined code options you may use the Ignite Sandbox based on the Java Sandbox model.
 
-You can read about the Java Sandbox model link:https://docs.oracle.com/en/java/javase/11/security/java-se-platform-security-architecture.html#GUID-C203D80F-C730-45C3-AB95-D4E61FD6D89C[here].
+You can read about the Java Sandbox model link:https://docs.oracle.com/en/java/javase/11/security/java-se-platform-security-architecture.html#GUID-C203D80F-C730-45C3-AB95-D4E61FD6D89C[here,window=_blank].
 
-There are a few conditions to run user-defined code with restrictions:
+== The Ignite Sandbox activation
 
-- link:/security/authentication[GridSecurityProcessor] is installed;
-- the method _GridSecurityProcessor#sandboxEnabled_ returns true;
-- link:https://docs.oracle.com/javase/8/docs/technotes/guides/security/spec/security-spec.doc6.html#a19349[SecurityManager] is installed.
+There are a few conditions to activate the Ignite Sandbox.
 
+- The first condition is _the SecurityManager must be installed_.
+
+Due to the Ignite Sandbox is based on the Java Sandbox model, you need to have installed the SecurityManager that is an important part of that model.
+The SecurityManager is responsible for checking the security policy currently in effect and perform access control checks.
+A security manager is not automatically installed when an application is running. If you run Ignite as a separate application,
+you must invoke the Java Virtual Machine with the `-Djava.security.manager` command-line argument (which sets the value of the `java.security.manager property`).
+There is also a `-Djava.security.policy` command-line argument whose usage determines what policy files are utilized.
+If you don't include `-Djava.security.policy` on the command line, then the policy files specified in the security properties file will be used.
+
+You can read more about link:https://docs.oracle.com/javase/8/docs/technotes/guides/security/spec/security-spec.doc6.html#a19349[Security Management,window=_blank].
+
+NOTE: It may be convenient adding the security manager and the policy command-line arguments to `{IGNITE-HOME}/bin/ignite.sh|ignite.bat` script.
+Read more about link:/quick-start/java#starting_a_node[starting a node].
+
+NOTE: The Ignite must have enough permissions to work correctly.
```

Comment:
'NOTE: Ignite should have enough permissions to work correctly.'
must is rude:)